tslearn/barycenters/dba.py

```diff
@@ -392,7 +412,7 @@ def _subgradient_update_barycenter(X, list_diag_v_k, list_w_k, weights_sum,
 
 
 def dtw_barycenter_averaging(X, barycenter_size=None, init_barycenter=None,
-                             max_iter=30, tol=1e-5, weights=None,
+                             max_iter=30, tol=1e-5, weights=None, n_treads=15,
```

Comment:
We use `n_jobs` everywhere as a parameter name, and `None` as its default, we should stick to the same convention here, I think.